test/testsuite.cpp

```diff
@@ -275,6 +275,21 @@ void TestFixture::complainMissingLib(const char * const libname) const
     missingLibs.insert(libname);
 }
 
+void TestFixture::printHelp()
+{
+    std::cout << "Testrunner - run Cppcheck tests\n"
+              "\n"
+              "Syntax:\n"
+              "    testrunner [OPTIONS] [TestClass::TestCase]\n"
+              "        testrunner runs all test cases.\n"
```

Comment:
I do not like that `<command> <comment>` is written on the same line with no clear separation.

Putting a `#` before the comment would be one way to solve it. Putting it on separate lines would be another:

    run all test cases:
    testrunner

Reply:
Agreed, the output was not good. I went with your second suggestion (with an added indentation of the commands). I'm not super happy about the output, but I tried adding a `#` but it didn't looked even worse.